src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -30,21 +30,21 @@ public void testCallCheck() {
 	@Test
 	public void testIsFull() {
 		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-		final int STACK_CAPACITY = 12;
-		for (int i = 0; i < STACK_CAPACITY; i++) {
+		final int NUMBERS_COUNT = 12;
+		for (int i = 0; i < NUMBERS_COUNT; i++) {
 			boolean result = rhymer.isFull();
 			Assert.assertEquals(false, result);
 			rhymer.countIn(888);
 		}
 
 		boolean result = rhymer.isFull();
-		Assert.assertEquals(true, result);
+		Assert.assertEquals(false, result); //zmieniono na full, ponieważ zmieniliśmy IntArrayAstack na IntLinkedList
 	}
 
 	@Test
 	public void testPeekaboo() {
 		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = -1;
+		final int EMPTY_STACK_VALUE = 0;
```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).